src/main/java/mate/academy/myJdbc/Main.java

```diff
@@ -0,0 +1,83 @@
+package mate.academy.myJdbc;
+
+import mate.academy.myJdbc.dao.DeveloperDao;
+import mate.academy.myJdbc.dao.DeveloperDaoImpl;
+import mate.academy.myJdbc.dao.ProjectDaoImpl;
+import mate.academy.myJdbc.model.Developer;
+import mate.academy.myJdbc.model.Project;
+import mate.academy.myJdbc.model.Skill;
+import mate.academy.myJdbc.service.DeveloperService;
+import mate.academy.myJdbc.service.DeveloperServiceImpl;
+import mate.academy.myJdbc.service.ProjectService;
+import mate.academy.myJdbc.service.ProjectServiceImpl;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.util.Set;
+
+public class Main {
+    public static void main(String[] args) {
+        Connection connection = MyConnectionUtil.getConnection();
+        DeveloperService developerService = new DeveloperServiceImpl(new DeveloperDaoImpl(connection));
+        ProjectService projectService = new ProjectServiceImpl(new ProjectDaoImpl(connection));
+
+//        System.out.println("Sum of developers salary for project Astra is: "
+//                + projectService.getSumSalaryForProject("Astra"));
```

Comment:
Commented code is not allowed. If you want test your code in main method - try to write examples that will be runnable on any workstations